com.unity.ml-agents/Runtime/EngineParameters.cs

```diff
@@ -0,0 +1,35 @@
+using MLAgents.SideChannels;
+using UnityEngine;
+
+namespace MLAgents
+{
+    /// <summary>
+    /// A container for the Engine Settings that can be specified at the start of
+    /// training. These Engine settings are set once at the start of training based on the
+    /// training configuration. The actual Engine settings applied may be slightly different
+    /// (e.g. TimeScale is clamped). This class enables the retrieval of the final settings. Note
+    /// that if the Engine settings are directly changed anywhere in your Project, then the values
+    /// returned here may not be reflective of the actual Engine settings.
+    /// </summary>
+    internal sealed class EngineParameters
```

Comment:
It feels like this class has no purpose now, since it's internal. Should we just remove it, and move the EngineConfigurationChannel/RegisterSideChannel setup to Academy?

Reply:
We can do that yes. This is without change to the API so we can hold it off or I can do it now. Any preference ?

Reply:
Then again, it's internal so we can remove it if we want.

Reply:
Oops, I was going to say do it later, but looks like you already did.

Reply:
The purpose of EngineParameters was just to contain the registering/unregistering.

Reply:
Fine to not have it, just wanted to clarify intentions.

Reply:
Yeah, I think it had a purpose originally, but we removed so much that it wasn't worth keeping around just to handle registration.